embedded-aerospike/src/test/java/com/playtika/test/aerospike/EmbeddedAerospikeBootstrapConfigurationTest.java

```diff
@@ -23,18 +23,18 @@
  */
 package com.playtika.test.aerospike;
 
+import static com.playtika.test.aerospike.AerospikeProperties.AEROSPIKE_BEAN_NAME;
```

Comment:
we use default intellij import order, where static imports are below non-static

Reply:
u may reimport for whole project using default intellij formatter

Reply:
Yeah, I know what you mean. I have a custom formatter that did this. I'll configure a default formatter for this project.